README.md

```diff
@@ -4,7 +4,7 @@
 [![License](https://img.shields.io/badge/license-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Build status](https://circleci.com/gh/getodk/central-backend.svg?style=shield)](https://circleci.com/gh/getodk/central-backend)
 
-ODK Central Backend is a minimal [ODK](https://getodk.org/) server based on Node.js and Postgres. It is currently under development.
+ODK Central Backend is the API server for [ODK](https://getodk.org/). It's built with Node.js and Postgres.
```

Comment:
Making a couple of quick cosmetic changes while here, hopefully that's ok! For better or worse, I think Central stopped being minimal a little while ago!

We used to use the "It is currently under development" language when we had some dead repos but now those have been removed.